### ISSUE TITLE
Include CSS content in accessibleText

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3767]
In axe-core 4.5 we deprecated support for IE11. Part of that means we should no longer be failing elements that have their accessible name created from the CSS content property. To do so, axe-core's accessibleText calculation will need to include CSS content. Something like the following should pass the `button-name` rule:

```html
<style> button:before { Content: "Hello world" } </style>
<button></button>
```

**Comment by scottaohara** [comment]
This came up today in a discussion, where a button which received its name from CSS content was being flagged as if it lacked a name.

This particular situation was that the CSS content in this example was simply the use of the "?" character.

e.g., 
```
<style>
.text::before {
  content: '?';
}
</style>
<button class=test></button>
```

This resulted in the button having the accessible name of "?", just like if someone had done the following:
```
<button>?</button>
```

Where the former is presently failing the automated scan, but the latter is not.

The "fun" thing about this particular situation though is that NVDA and JAWS with default settings treat both treatments of this button as if they have no name, likely do to their own verbosity defaults for how to announce (or not in this case) punctuation.  

I dunno if in the work that's going on here, maybe a best practice flag can also be raised for situations like this?  e.g., there is technically a name here, but default screen reader settings might not expose it, so, be careful?  shrug...

**Comment by Sandrakay49** [comment]
I want copy of everything  I've had enough of them doing this
